src/_pytest/fixtures.py

```diff
@@ -660,12 +660,6 @@ def addfinalizer(self, finalizer):
         self._fixturedef.addfinalizer(finalizer)
 
 
-class ScopeMismatchError(Exception):
```

Comment:
this one should be in actual use, what happened there

Reply:
3ed3e51997b30f2d739515c7daa33f086379e182 was the last usage removed

Reply:
yikes, its bad that this was missed, but now its way too late ^^